api/resource/resource_test.go

```diff
@@ -103,14 +135,17 @@ func TestResourceId(t *testing.T) {
 }
 
 func TestDeepCopy(t *testing.T) {
-	r := factory.FromMap(
+	r, err := factory.FromMap(
 		map[string]interface{}{
 			"apiVersion": "apps/v1",
 			"kind":       "Deployment",
 			"metadata": map[string]interface{}{
 				"name": "pooh",
 			},
 		})
+	if err != nil {
+		t.Fatal(err)
```

Comment:
Could you add some messages to describe what happens with fatal functions?

Reply:
@koba1t 
Oh, sorry. I will fix that